pallets/rad-claims/README.md

```diff
@@ -0,0 +1,90 @@
+# Rad Claims Pallet
+
+<!-- TOC -->
+
+- [Rad Claims Pallet](#rad-claims-pallet)
+    - [Overview](#overview)
+    - [Pallet Usage](#pallet-usage)
+        - [Add the Pallet to your Parachain Runtime](#add-the-pallet-to-your-parachain-runtime)
+        - [Configure the Pallet](#configure-the-pallet)
+    - [Pallet Documentation](#pallet-documentation)
+    - [References](#references)
+    - [License](#license)
+
+<!-- /TOC -->
+
+## Overview
+
+This [FRAME](https://substrate.dev/docs/en/knowledgebase/runtime/frame) pallet 
+provides functionalities for processing RAD token rewarding claims.
+
+## Pallet Usage
+
+### Add the Pallet to your Parachain Runtime
+
+In order to add this pallet to your runtime, you should add the following lines
+to your parachain's main `Cargo.toml` file:
+
+```toml
+# -- snip --
+
+[dependencies.pallet-rad-claims]            # <-- Add the new dependency
+default_features = false
+git = 'https://github.com/centrifuge-chain/pallet-rad-claims.git'
```

Comment:
Not sure what is this for but might as well use the complete url
```suggestion
git = 'https://github.com/centrifuge/centrifuge-chain/pallet-rad-claims.git'
```

Reply:
Just an example on how to use the pallet (if the latter would be in its own repo). I do not wanted to put a filesystem path instead of a git URL before we shall refactor our various runtimes (as discussed last time).